practice/src/index.html

```diff
@@ -12,149 +18,161 @@
-                <use xlink:href="./assets/images/icons/icons.svg#home-icon" />
+        <div class="homepage-main">
+          <!-- Card Add -->
+          <div class="card card-add-product">
+            <a href="javascrip:void(0)" class="card-add-title">
+              <svg width="15" height="15">
+                <use xlink:href="./assets/images/icons/icons.svg#add-icon" />
               </svg>
+              Add new dish
             </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#discount-icon"
-                />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use xlink:href="./assets/images/icons/icons.svg#graph-icon" />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#message-icon"
-                />
-              </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#notification-icon"
-                />
+          </div>
+          <!-- Card Products-->
+          <div class="card card-product">
+            <div class="card-header">
+              <svg width="15" height="15">
+                <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
               </svg>
-            </a>
-          </li>
-          <li class="nav-item">
-            <a href="javascript:void(0)" class="nav-link">
-              <svg>
-                <use
-                  xlink:href="./assets/images/icons/icons.svg#setting-icon"
+            </div>
+            <div>
+              <figure class="card-image">
+                <img
+                  src="./assets/images/food.png"
+                  alt="food image"
+                  class="card-item"
                 />
-              </svg>
-            </a>
-          </li>
-        </ul>
-      </div>
-      <div class="sidebar-footer">
-        <a href="javascript:void(0)" class="sidebar-footer-link">
-          <svg>
-            <use xlink:href="./assets/images/icons/icons.svg#logout-icon" />
-          </svg>
-        </a>
-      </div>
-    </aside>
+              </figure>
+              <div class="card-text">
+                <p class="tertiary-title card-name">
+                  Spicy seasoned seafood noodles
+                </p>
+                <div class="main-desc card-desc">
+                  <p>$ 2.29</p>
+                  <span class="circle">&#8729;</span>
```

Comment:
You only use padding left and right for this instead of use display: flex for .card-desc